CONTRIBUTING.md

```diff
@@ -88,7 +88,7 @@ $> swift --version
 Apple Swift version 5.3
 ```
 
-Note: Alternatively use tools like [swiftenv](https://github.com/kylef/swiftenv) that help manage toolchains versions.
+Note: Alternatively use tools like [swiftly](https://github.com/swiftlang/swiftly) that help manage toolchains versions.
```

Comment:
IMO linking to the repository would make sense for contributors to swiftly, but not for users. This should link to user docs at https://www.swift.org/swiftly/documentation/swiftlydocs/ instead.